src/main/java/com/mraof/minestuck/data/recipe/MinestuckRecipeProvider.java

```diff
@@ -271,6 +271,50 @@ protected void buildRecipes(Consumer<FinishedRecipe> recipeBuilder)
+		ShapedRecipeBuilder.shaped(RecipeCategory.BUILDING_BLOCKS, MSBlocks.POLISHED_CRUXITE_SLAB.get(), 6).define('#', MSBlocks.POLISHED_CRUXITE_BLOCK.get()).pattern("###").group("cruxite_blocks").unlockedBy("has_raw_cruxite", has(MSItems.RAW_CRUXITE.get())).save(recipeBuilder);
+		ShapedRecipeBuilder.shaped(RecipeCategory.BUILDING_BLOCKS, MSBlocks.POLISHED_CRUXITE_WALL.get(), 6).define('#', MSBlocks.POLISHED_CRUXITE_BLOCK.get()).pattern("###").pattern("###").group("cruxite_blocks").unlockedBy("has_raw_cruxite", has(MSItems.RAW_CRUXITE.get())).save(recipeBuilder);
+		ShapedRecipeBuilder.shaped(RecipeCategory.BUILDING_BLOCKS, MSBlocks.CRUXITE_BRICKS.get(), 4).define('#', MSBlocks.CRUXITE_BLOCK.get()).pattern("## ").pattern("## ").group("cruxite_blocks").unlockedBy("has_raw_cruxite", has(MSItems.RAW_CRUXITE.get())).save(recipeBuilder);
+		ShapedRecipeBuilder.shaped(RecipeCategory.BUILDING_BLOCKS, MSBlocks.CRUXITE_BRICK_STAIRS.get(), 4).define('#', MSBlocks.CRUXITE_BRICKS.get()).pattern("#  ").pattern("## ").pattern("###").group("cruxite_blocks").unlockedBy("has_raw_cruxite", has(MSItems.RAW_CRUXITE.get())).save(recipeBuilder);
+		ShapedRecipeBuilder.shaped(RecipeCategory.BUILDING_BLOCKS, MSBlocks.CRUXITE_BRICK_SLAB.get(), 6).define('#', MSBlocks.CRUXITE_BRICKS.get()).pattern("###").group("cruxite_blocks").unlockedBy("has_raw_cruxite", has(MSItems.RAW_CRUXITE.get())).save(recipeBuilder);
+		ShapedRecipeBuilder.shaped(RecipeCategory.BUILDING_BLOCKS, MSBlocks.CRUXITE_BRICK_WALL.get(), 6).define('#', MSBlocks.CRUXITE_BRICKS.get()).pattern("###").pattern("###").group("cruxite_blocks").unlockedBy("has_raw_cruxite", has(MSItems.RAW_CRUXITE.get())).save(recipeBuilder);
+		SimpleCookingRecipeBuilder.smelting(Ingredient.of(MSBlocks.CRUXITE_BLOCK.get()), RecipeCategory.BUILDING_BLOCKS, MSBlocks.SMOOTH_CRUXITE_BLOCK.get(), 0.1f, 200).group("cruxite_blocks").unlockedBy("has_raw_cruxite", has(MSItems.RAW_CRUXITE.get())).save(recipeBuilder);
+		ShapedRecipeBuilder.shaped(RecipeCategory.BUILDING_BLOCKS, MSBlocks.CHISELED_CRUXITE_BLOCK.get(), 4).define('#', MSBlocks.CRUXITE_BRICKS.get()).pattern("## ").pattern("## ").group("cruxite_blocks").unlockedBy("has_raw_cruxite", has(MSItems.RAW_CRUXITE.get())).save(recipeBuilder);
+		ShapedRecipeBuilder.shaped(RecipeCategory.BUILDING_BLOCKS, MSBlocks.CRUXITE_PILLAR.get(), 2).define('#', MSBlocks.SMOOTH_CRUXITE_BLOCK.get()).pattern("#  ").pattern("#  ").group("cruxite_blocks").unlockedBy("has_raw_cruxite", has(MSItems.RAW_CRUXITE.get())).save(recipeBuilder);
+		ShapedRecipeBuilder.shaped(RecipeCategory.BUILDING_BLOCKS, MSBlocks.CRUXITE_LAMP.get(), 1).define('#', MSItems.RAW_CRUXITE.get()).define('%', Items.TORCH).pattern("###").pattern("#%#").pattern("###").group("cruxite_blocks").unlockedBy("has_raw_cruxite", has(MSItems.RAW_CRUXITE.get())).save(recipeBuilder);
```

Comment:
Given the capacity to turn it off and on it feels from my perspective like it should use an alternative light source. Perhaps glowstone dust or uranium? I also think it would make sense to use the other forms of cruxite given the sculpted look of the texture! Feel free to make an argument if you want it to stay that way, I see no major game design issues with keeping it this way

Reply:
Hmm I do see what you're saying. My goal was to align it with the lantern, which needs 8 iron nuggets ( -> 1 iron ingot -> 1 iron ore). To account for the ability to turn it on and off, 8 cruxite ore seemed both a logical step up from the lantern and a fair price, considering that the on/off is purely decorative and has to be manually toggled so you can't use it like a redstone lamp. 

I wouldn't be opposed to changing it to 4 uranium + 4 cruxite + torch though. Or maybe 8 cruxite + 1 lantern?